calculator.rb

```diff
@@ -0,0 +1,58 @@
+# prints question
+puts "Ready to do some math?"
+# deletes any spaces after the answer provided by the user
+answer = gets.chomp
+
+# ".downcase" takes user input makes it all lowercase
+#  this accounts for any uppercase/lowercase form the word "yes" or "no"
+# depending on user input, if one of the following is true it will prin the line below
+if answer.downcase == "yes"
+  puts "Alrighty then!"
+elsif answer.downcase =="no"
```

Comment:
Add a space after `==` in your `elsif` for consistency with your `if` conditional